aspnetcore/migration/22-to-30.md

```diff
@@ -42,3 +42,21 @@ This article explains how to update an existing ASP.NET Core 2.2 project to ASP.
 * Some assemblies were removed from `Microsoft.AspNetCore.App` between 2.x and 3.0. You may need to add `<PackageReference>` items if you're using APIs from packages listed in [aspnet/AspNetCore #3755](https://github.com/aspnet/AspNetCore/issues/3755)
 
   For example, `Microsoft.EntityFrameworkCore` and `System.Data.SqlClient` are no longer part of `Microsoft.AspNetCore.App`. The list of assemblies shipping in `Microsoft.AspNetCore.App` hasn't been finalized yet and will change before 3.0 RTM.
+
+* Add [Json.NET support](#json)
+
+<a name="json"></a>
+
+## Json.NET support
+
+As part of the work to [improve the ASP.NET Core shared framework](https://blogs.msdn.microsoft.com/webdev/2018/10/29/a-first-look-at-changes-coming-in-asp-net-core-3-0/), [Json.NET](https://www.newtonsoft.com/json/help/html/Introduction.htm) has been removed from the ASP.NET Core shared framework.
+
+To use Json.NET in an ASP.NET Core 3.0 project:
+
+- Add a package reference to [Microsoft.AspNetCore.Mvc.NewtonsoftJson](https://nuget.org/packages/Microsoft.AspNetCore.Mvc.NewtonsoftJson)
+- Update `ConfigureServices` to call `AddNewtonsoftJson()`.
+
+    ```csharp
+    services.AddMvc()
+        .AddNewtonsoftJson();
```

Comment:
You could mention that Newtonsoft settings can be set with AddNewtonsoftJson:

https://github.com/aspnet/AspNetCore/blob/cdc66c5f98f73f0763573db4274b6763fbc6788d/src/Mvc/benchmarkapps/BasicApi/Startup.cs#L128

SignalR also uses Json.NET and has added the `AddNewtonsoftJsonProtocol` method. I'm not fimilar with the steps to upgrade SignalR

https://github.com/aspnet/AspNetCore/blob/3cd5054eb5d8851b25881cd05cccb931aaa022e6/src/SignalR/common/Protocols.NewtonsoftJson/src/NewtonsoftJsonProtocolDependencyInjectionExtensions.cs